packages/rest/src/rest.server.ts

```diff
@@ -716,6 +717,16 @@ export class RestServer extends Context implements Server, HttpServerLike {
     this.sequence(SequenceFromFunction);
   }
 
+  /**
+   * Bind a body parser to the server context
+   * @param bodyParserClass
+   */
+  bodyParser(bodyParserClass: Constructor<BodyParser>) {
+    this.bind(`${RestBindings.REQUEST_BODY_PARSER}.${bodyParserClass.name}`)
```

Comment:
Let's extract this logic into a public helper function that 3rd-party components can use too.

```ts
bodyParser(bodyParserClass: Constructor<BodyParser>) {
  this.add(createBodyParserBinding(bodyParserClass));
}
```